src/runtime/acknowledged_message.rs

```diff
@@ -21,8 +21,8 @@ impl<M, R> AcknowledgedMessage<M, R> {
     }
 
     /// Get the message.
-    pub(crate) fn message(&self) -> &M {
-        &self.message
+    pub(crate) fn message(self) -> M {
```

Comment:
Now that this takes ownership, can we name it `into_message`?

Reply:
Updated